### PR TITLE
fix: import profile task keeps running on failure

### PR DESCRIPTION
**Pull request by connoratrug**
### What are the main changes you did
- catch errors that are thrown during profile loading 
  -  also catch ( and handle ) error thrown during exception handling 

### How to test
- create a profile import that contains a error, for example use a yml file that points to a non existent file 
- without these changes the loading would fail the the taks would keep running, without reporting the error back to the ui. with these changes the error thrown by the sub task cancellation gets caught and handled, the user is then notified via the ui

Note: this is a workaround for issues related to the taks management, fixing these would require to somewhat re-architect the task system (out of scope of this pr). Similar issues as the one addressed remain for other users of the task system and may be 'solved' by a similar work around.    

### Checklist
- [ ] updated docs in case of new feature
- [ ] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

### CODE REVIEW

File: backend/molgenis-emx2-datamodels/src/main/java/org/molgenis/emx2/io/ImportProfileTask.java [hunk 1]
```diff
@@ -42,9 +42,15 @@ public void run() {
...
-      commitTask.completeWithError("Commit failed: " + e.getMessage());
-      this.completeWithError(e.getMessage());
-      throw (e);
+      try {
+        commitTask.completeWithError("CommitTask failed: " + e.getMessage());
+      } catch (MolgenisException e2) {
+        try {
+          this.completeWithError("ImportProfileTask  failed: " + e2.getMessage());
+        } catch (Exception e3) {
+          throw (e3);
```
[the remark this answers is not in the document]
Comment (harmbrugge) [reply backend/molgenis-emx2-datamodels/src/main/java/org/molgenis/emx2/io/ImportProfileTask.java:51]: Interesting

**Review by harmbrugge — APPROVED** [review]

File: backend/molgenis-emx2-datamodels/src/main/java/org/molgenis/emx2/io/ImportProfileTask.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (mswertz) [reply backend/molgenis-emx2-datamodels/src/main/java/org/molgenis/emx2/io/ImportProfileTask.java:51]: hmmm, ugly but lets not worry?

**Review by mswertz — APPROVED** [review]

**Comment by mswertz** [comment]
formally we should include a test that requires this exception to be caught?

**Comment by mswertz** [comment]
to reach codecov

**Pull request merged**